src/pages/NewSighting.js

```diff
@@ -0,0 +1,124 @@
+import { Link } from "react-router-dom";
+import { useEffect, useState } from "react";
+import Select from "react-select";
+import { BACKEND_URL } from "../constants";
+import "../App.css";
+
+const NewSighting = () => {
+  const [categoryOptions, setCategoryOptions] = useState([]);
+  const [inputFields, setInputfields] = useState({
+    date: "",
+    location: "",
+    notes: "",
+  });
+  const [selectFields, setSelectFields] = useState([]);
+
+  const fetchCategories = async () => {
+    const fetchedCategories = await fetch(BACKEND_URL + "/categories/", {
+      method: "get",
+    });
+    const fetchedCategoriesJson = await fetchedCategories.json();
+
+    const newCategoryOptions = fetchedCategoriesJson.map((response) => ({
+      value: response.id,
+      label: response.categoryName,
+    }));
+
+    setCategoryOptions(newCategoryOptions);
+    console.log(categoryOptions);
+  };
+
+  useEffect(() => {
+    fetchCategories();
+  }, []);
+
+  // Make text black in Select field
+  const selectFieldStyles = {
+    option: (provided) => ({
+      ...provided,
+      color: "black",
+    }),
+  };
+
+  const handleChange = (e, field) => {
+    let newFields = { ...inputFields };
+    newFields[field] = e.target.value;
+    setInputfields(newFields);
+  };
+  const handleSelectChange = (e) => {
+    const categoriesArr = e.map((obj) => obj.value);
+    console.log(categoriesArr);
+    setSelectFields(e);
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const addNewSighting = async () => {
+      const categoriesArr = selectFields.map((obj) => obj.value);
+      await fetch(BACKEND_URL + "/sightings/", {
+        method: "post",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ input: inputFields, categories: categoriesArr }),
+      })
+        .then((response) => response.json())
+        .then((data) => console.log(data));
```

Comment:
why use promise and all if you just console.log in the end? Usually you would need to update the list of sightings after creating a new one